roles/parse_leapp_report/tasks/main.yml

```
@@ -21,10 +21,12 @@
 - name: Check for inhibitors
   ansible.builtin.set_fact:
     upgrade_inhibited: true
+    leapp_inhibitors: "{{ leapp_inhibitors | default([]) + [item] }}"
   when:
     - item.key not in leapp_known_inhibitors | default([], true)
     - (leapp_high_sev_as_inhibitors | default(false, true) | bool and item.severity == 'high') or
-      'inhibitor' in item.flags
+      'inhibitor' in item.groups or
+      'error' in item.groups
```

Hit a `'dict object' has no attribute 'groups'` failure with this task testing on RHEL8 because one of the finding entries did not have a `groups` key. I asked my Leapp friends and was referred [here](https://github.com/oamg/schema-test/blob/d4d8a14cc998e9ba67f3f1ec8af55a2786798a6e/report-schema-v120.json#L181). The `groups` key is not required/mandatory, so when it's empty, it's not part of the entry. I think this would fix it... 
```suggestion
      'inhibitor' in item.groups | default([], true) or
      'error' in item.groups | default([], true)
```